libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -523,7 +523,7 @@ class YulUtilFunctions
 	/// @returns the name of a function that will clear given storage slot
 	/// starting with given offset until the end of the slot
 	/// signature: (slot, offset)
-	std::string partialClearStorageSlotFunction();
+	std::string partialClearStorageSlotFunction(bool _leftAlignedType);
```

Comment:
Maybe it would be better to use two different functions for the two alignments?

Reply:
That is true, we can call explicitly the one we want instead of having parameter

Reply:
Acutally the main purpose for the current function is to clear data in packed storage slots. If the alignment is different, then we should probably also use a different function name to clarify this distinction.

Reply:
I'll do this later, not needed in this PR